src/index.html

```diff
@@ -17,11 +20,378 @@
+            <a
+              class="nav__link nav__link--hover"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="grid-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="stars stars--active">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-number">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="stars stars--active">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-number">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
```

Comment:
The 'buy' class is used for the Buy button. Ensure that this class is styled appropriately in your CSS, and consider whether additional classes or attributes are needed for functionality or styling.